### PR TITLE
Updates to models/fates/R/model2netcdf.FATES.R

### PR DESCRIPTION
**Pull request by serbinsh**
Updates to models/fates/R/model2netcdf.FATES.R to begin expanding outputs, still much work to be done

## Description
Updating the number of outputs, adding in longname

## Motivation and Context
Enhance outputs from FATES simulations

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [ ] When possible
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ X] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [X ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by serbinsh** [comment]
Additional info on status (from an email exchange):


Indeed, we have integrated FATES into PEcAn with basic support for UQ/ensemble runs for a single tropical PFT.  Both Mike and I (and others) spent a lot of time on this, on and off, over the period of initial FATES evolution and development.  That said, on my TODO is to bring FATES support in PEcAn up to match the latest version that will become "public" shortly.  This includes support for the expanded PFTs in FATES and making sure parameter names provided by PEcAn match FATES, with likely a few other small tweaks needed (e.g. cloning runs instead of re-building for each ensemble member).  I put this on hold for a couple of reasons, 1) I was very busy with field work over the summer (and could not keep up with the developments during that time) and 2) FATES was still undergoing a lot of modifications, parameter name changes, etc that made it pretty hard to keep up.  That said, it looks like things are stabilizing again so this is something I have actually put on my NGEE-Tropics tasks list for the next month.

- Next steps for FATES to increase functionality in PEcAn include the option to initialize from stand history, similar to ED2, testing the ability to run with the expanded PFTs at appropriate locations, and finally running FATES with various met drivers available in PEcAn; that functionality needs to be well tested and brought back in where necessary, and which should be possible.

- Finally, at present PEcAn supports CTSM-FATES (CTSM/CLM as host land mode) and I anticipate this moving forward until we can test / incorporate ELM (E3SM land model host).  That is farther behind because it had been more difficult to get access, but that has since changed.

- To summarize, FATES is "supported" in PEcAn but I (we) would like to / need to work on checking through and fixing and new bugs, etc as well as adding in a bit more functionality.  Again, this is something that is in the works but I (we) are happy to discuss in more detail, etc.

**Comment by serbinsh** [comment]
In support of issue #1008

**Comment by dlebauer** [comment]
Here is an example CLM output that can be used for development: https://drive.google.com/open?id=17V3WPoHs8taUD0bWteEy9sgd9yrOzMGP

**Comment by tonygardella** [comment]
@serbinsh I'm gonna merge this so I can build off of it for the general ctsm function.
